backend/src/models/vulnerability.ts

```diff
@@ -136,6 +136,7 @@ export class Vulnerability extends BaseEntity {
     date: Date;
   }[];
 
+  //Stores vulnerability data in a json object associated with a specific scan type
```

Comment:
Can you format the comment like how the other comments for properties are formatted?

i.e., like

```
  /** Determines the source of the vulnerability, i.e., which
   * logic caused it to be created.
   */
```

Reply:
Something like this?

/** Stores structured data related to a vulnerability, e.g., a list of 
   * emails found in breaches for a domain
   */